content/chapters/chapter-8/index.tsx

```diff
@@ -22,8 +16,18 @@ export const metadata = {
   slug: 'chapter-8',
   title: 'chapter_eight.title',
   image: '/assets/images/chapter-8-cover.jpg',
-  intros: [],
-  lessons: [],
-  outros: [],
-  challenges: [],
+  theme: 'bg-gradient-to-br from-[#0A1952] to-[#1B3D98]',
+  intros: ['intro-1', 'intro-2', 'intro-3'],
+  lessons: [
+    'building-blocks-1',
+    'building-blocks-2',
+    'building-blocks-3',
+    'building-blocks-4',
+    'building-blocks-5',
+    'building-blocks-6',
+    'building-blocks-7',
+    'building-blocks-8',
+  ],
+  outros: ['outro-1'],
+  challenges: ['building-blocks-3'],
```

Comment:
the challenge here would still start at 'building-blocks-1' even though they are just text pages